src/components/ComposerQuill/sanitize.js

```diff
@@ -24,6 +24,10 @@ function sanitizerPlugin(md) {
           }
         });
       }
+      if (type === 'html_block') {
+        // eslint-disable-next-line no-param-reassign
+        token.content = token.content.replace(/</g, '&lt;').replace(/(?!^)>/gm, '&gt;');
```

Comment:
@phatv can you weigh in on this?

Reply:
This looks fine to me. `html_block` might have element tags in it that we don't want to actually display as elements but instead as the carets.

But would you mind sharing what the text was for it to come in as html_block?

Reply:
it is ``<iframe onload=alert(document.cookie) %00 src="<h1>Injected H1</h1>"%00>`` that will come inside html_block .